src/main/resources/application.yaml

```
@@ -1,76 +1,80 @@
+  profiles:
+    active: local
+  servlet:
+    multipart:
+      max-file-size: 10MB
+      max-request-size: 10MB
+
+tour-api:
+  base-url: https://apis.data.go.kr
+  mobile-os: WEB
+  mobile-app: LocalUp
+    # 한국관광공사_국문 관광정보 서비스_GW
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)